topsail/ansible_default_config.py

```diff
@@ -35,7 +35,17 @@ def _generate_config(component):
 
     classname = component.__qualname__.partition(".")[0].lower()
 
-    dest = pathlib.Path("roles") / classname / component.ansible_role / "defaults" / "main" / "config.yml"
+    if classname == component.ansible_role:
+
+        if classname == 'fetch_external_test':
+            # TODO:FIXME: Check this convention defaults/main.yml vs defaults/main/config.yml
+            # TODO:FIXME: Why nesting folders for the roles definitions?
```

Comment:
if you want, you can make a PR that 
* deletes all the `defaults/main/config.yml` files, 
* updates `generate_ansible_default_settings` to remove `/main/`
* includes the new `default/config.yaml` files 
we'll see if anything goes wrong with the test.

that's a trial by combat :D